src/main/kotlin/com/trivigenai/repository/GenAIRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.trivigenai.repository
+
+import com.trivigenai.models.Round
+
+interface GenAIRepository {
+    fun generateTrivia(
+        topic: String?,
+        retries: Int = 3
```

Comment:
Why the retries is a parameter? Isn't better to create value in GenAIRepositoryImpl class?

Reply:
Thanks. I was using a recursive approach earlier and forgot to remove it as a parameter.